piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -97,9 +141,17 @@
   private final HashSet<String> treatedMethods = new HashSet<String>();
   private final HashSet<String> controlMethods = new HashSet<String>();
   private final HashSet<String> deleteMethods = new HashSet<String>();
+  private final HashSet<String> treatmentGroupMethods = new HashSet<String>();
   private final HashSet<String> handledAnnotations = new HashSet<String>();
   private String linkURL;
 
+  /** State used to track usage counts and delete corresponding declarations if needed. */
+  private TreePath cuPath = null;
+
+  private boolean countsCollected = false;
+  private ImmutableMap<Symbol, UsageCounter.CounterData> usageCounts = null;
+  private Map<Symbol, Integer> deletedUsages = null;
+
   /**
    * Copied from NullAway comment. Error Prone requires us to have an empty constructor for each
```

Comment:
This line can be deleted :)

Reply:
Seems useful for documenting why we have that constructor, though. I can remove the reference from NullAway if you want, but the comment itself seems useful.